api/models/trial.js

```diff
@@ -133,14 +133,25 @@ const Trial = BaseModel.extend({
                                       .toJSON()
                                       .map((publication) => publication.source);
       const documentsSources = this.related('documents')
-                                 .toJSON()
-                                 .map((document) => document.source);
+                                   .toJSON()
+                                   .map((document) => document.source);
       const recordsSources = this.related('records')
                                  .toJSON()
                                  .map((record) => record.source);
+      const robSources = this.related('risks_of_bias')
+                             .toJSON()
+                             .map((rob) => {
+                               return {
+                                 id: rob.source_id,
+                                 name: "Cochrane Schizophrenia group",
+                                 source_url: rob.source_url,
+                               };
+                             });
```

Comment:
We do have the source in our database, see https://github.com/opentrials/processors/blob/master/processors/cochrane_reviews/extractors.py#L45-L51. So this is the same as the other entities (`record`, `documents`, etc.).

While you're there, as you changed the `documentsSources` anyway, please change to use `doc` instead of `document`, as `document` is a reserved keyword in JavaScript.